src/sets.jl

```diff
@@ -1188,7 +1220,12 @@ products we have
 """
 abstract type AbstractSymmetricMatrixSetTriangle <: AbstractVectorSet end
 
-function dimension(set::AbstractSymmetricMatrixSetTriangle)
+function dimension(
+    set::Union{
+        AbstractSymmetricMatrixSetTriangle,
+        ScaledPositiveSemidefiniteConeTriangle,
```

Comment:
Shouldn't `ScaledPositiveSemidefiniteConeTriangle` be `<:AbstractSymmetricMatrixSetTriangle` instead of `<:AbstractVectorSet`?

Or am I forgetting some triviality.